src/app/mod.rs

```diff
@@ -300,32 +301,75 @@ pub async fn run_app(cli: cli::DftCli, state: state::AppState<'_>) -> Result<()>
     info!("Running app with state: {:?}", state);
     let mut app = App::new(state, cli.clone());
 
-    match &cli.command {
```

Comment:
this just removes a seemingly unecessary level of wrapping of Clap argument parsing